src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAYCOST = 40;
+  const WEEKLENGTH = 7;
+  const WEEKOFF = 50;
+  const THREEDAYS = 3;
+  const THREEDAYSOFF = 20;
```

Comment:
please check once more the checklist file, you have examples there, use snake_case to make your constants better readable and also a good practice it to have them outside of the function, in the beginning of the file